config/src/client.rs

```diff
@@ -79,10 +81,12 @@ pub struct Configuration {
     /// Status URL.
     pub torii_telemetry_url: Url,
     /// Proposed transaction TTL in milliseconds.
-    pub transaction_time_to_live_ms: u64,
+    pub transaction_time_to_live_ms: Option<NonZeroU64>,
     /// Transaction status wait timeout in milliseconds.
     pub transaction_status_timeout_ms: u64,
     /// The limits to which transactions must adhere to
+    // NOTE: If you want this functionality, implement it in the app manually
+    #[deprecated(note = "This parameter is not used and takes no effect")]
```

Comment:
```suggestion
    #[deprecated(note = "This parameter is not used and takes no effect. If you want this functionality, implement it in the app manually.")]
```

Saying to user what should they do with a deprecation helps.